web/styles/zulip.css

```diff
@@ -938,7 +938,6 @@ div.focused-message-list {
 
 #move_topic_modal select {
     width: auto;
-    margin-bottom: 10px;
     max-width: 100%;
 }
```

Comment:
This should probably move to `web/styles/popovers.css` and also be on `.modal_select`, not `select`.